lib/screens/play/card/card_display.dart

```diff
@@ -6,10 +6,10 @@ import './card_header.dart';
 import '../../../model/card/card.dart' as Model;
 
 class CardDisplay extends StatelessWidget {
-  final Model.Card card;
+  final Model.Card? card;
```

Comment:
Would we ever have a `CardDisplay` with no card? I guess maybe this is a knock-on effect of `currentCard` being nullable? Feels like another one to revisit :smile: